src/borrower/feed/navfeed.sol

```diff
@@ -37,8 +38,10 @@ contract NAVFeed is BaseNFTFeed, Interest, Buckets {
         uint percentage;
     }
 
-    uint public discountRate;
-    uint public maxDays;
+    // default 3% a day
+    uint public discountRate = uint(1000000342100000000000000000);
```

Comment:
should we have the default values set in the contract?

Reply:
I will change that. It was just my quick fix to migrate the tests